warp/src/main/res/drawable/planeland.xml

```diff
@@ -0,0 +1,20 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
for language consistency, nitpicking and having something to say, I think this should be called `planelanding` 😆